format-diff.py

```
@@ -29,6 +29,22 @@ def run_cmd(cmd, show_cmd=False):
+def try_find_clang_format(exec_path):
+    candidates = ['clang-format-13']
+    if exec_path is not None:
+        candidates.insert(0, exec_path)
+    for c in candidates:
+        if which(c) is not None:
+            return c
+    return candidates[-1]
+
+
+def get_clang_format_version(clang_format_cmd):
+    proc = subprocess.run([clang_format_cmd, '--version'], text=True, check=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)